src/dev/impl/DevToys/Views/Tools/EncodersDecoders/CertificateEncoderDecoder/CertificateEncoderDecoderToolPage.xaml

```diff
@@ -0,0 +1,86 @@
+﻿<Page
+    x:Class="DevToys.Views.Tools.CertificateEncoderDecoder.CertificateEncoderDecoderToolPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevToys.UI.Controls"
+    xmlns:toolkit="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required">
+
+    <Grid x:Name="RootGrid" RowSpacing="12" Margin="0,0,0,16">
+        <Grid.ChildrenTransitions>
+            <TransitionCollection>
+                <EntranceThemeTransition IsStaggeringEnabled="True" FromVerticalOffset="50"/>
+            </TransitionCollection>
+        </Grid.ChildrenTransitions>
+
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition MinWidth="100"/>
+            <ColumnDefinition Width="16"/>
+            <ColumnDefinition MinWidth="100"/>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition MinHeight="100"/>
+            <RowDefinition MinHeight="100"/>
+        </Grid.RowDefinitions>
+
+        <PasswordBox
```

Comment:
How about adding a top marging to the password box, just so it's vertically aligned with the text box of the Output?

![image](https://user-images.githubusercontent.com/3747805/234469842-0d77bfbf-7df4-47a7-b011-87e809989a7f.png)

By the way this is something I can make more automatic in DevToys 2.0

Reply:
Ah, great call that looks so much better:
![image](https://user-images.githubusercontent.com/74258557/234687614-bf0be033-777d-44aa-8054-6dfdd8e0e4bf.png)

I'd be curious to know what you're thinking for automating this.